lib/parsers/asm-parser.js

```diff
@@ -37,11 +37,11 @@ export class AsmParser extends AsmRegex {
         this.labelFindMips = /[$.A-Z_a-z][\w$.]*/g;
         this.mipsLabelDefinition = /^\$[\w$.]+:/;
         this.dataDefn = /^\s*\.(string|asciz|ascii|[1248]?byte|short|x?word|long|quad|value|zero)/;
-        this.fileFind = /^\s*\.file\s+(\d+)\s+"([^"]+)"(\s+"([^"]+)")?.*/;
+        this.fileFind = /^\s*\.(?:cv_)?file\s+(\d+)\s+"([^"]+)"(\s+"([^"]+)")?.*/;
         // Opcode expression here matches LLVM-style opcodes of the form `%blah = opcode`
         this.hasOpcodeRe = /^\s*(%[$.A-Z_a-z][\w$.]*\s*=\s*)?[A-Za-z]/;
         this.instructionRe = /^\s*[A-Za-z]+/;
-        this.identifierFindRe = /[$.@A-Z_a-z][\dA-z]*/g;
+        this.identifierFindRe = /[$.@A-Z_a-z]\w*/g;
```

Comment:
\w also includes underscore _, are there situations where labels are encoded this way?

Reply:
Matt asked for `[\dA-Z_a-z]` so I thought the underscore was wanted even if it did not show up in the original RE. Happy to remove it :)

Reply:
let me give it some thought

Reply:
ah A-z does actually include _ and some other stuff. Question is do we actually need it. And I can't really answer that question, let's keep it \w until it breaks something